packages/gatsby-theme-project-portal/src/layouts/SearchPageLayout.tsx

```diff
@@ -0,0 +1,185 @@
+    /* allow stop words (see https://github.com/olivernn/lunr.js/blob/master/lunr.js)
+     we have 'about' as one of the slugs, which spurred this decision
+    We can implement our own stopWordFilter:
+    var idx = lunr(function () {
+      // normal index definition
+      this.pipeline.after(lunr.stopWordFilter, myStopWordFilter)
+       this.pipeline.remove(lunr.stopWordFilter)
+    })
+     */
+    this.pipeline.remove(lunr.stopWordFilter)
+
+    documents.forEach(function (doc) {
+      this.add(doc)
+    }, this)
+  })
+
+  useEffect(() => {
+    if (searchQuery.length > 0) {
+      let searchResults = idx.search(searchQuery)
+      let results = []
+      searchResults.forEach(function (result) {
+        results.push(db[result.ref])
+      })
+      setQueryResults(searchResults)
+    } else {
+      setQueryResults([])
+    }
+  }, [searchQuery])
+  //const meta = useSiteMetadata()
+  return (
+    <div className="flex flex-col items-center justify-center align-middle text-center px-2">
+      <div className="container mx-auto mt-2 border-b-4 border-primary">
+        <SearchBar
+          id="siteSearch"
+          autoFocus={true}
+          label={"Search"}
+          placeholder={"Type to search pages..."}
+          onChange={(e) => setSearchQuery(e.target.value)}
+        />
+        <div className=" pt-2">
+          Number of found pages:
+          {queryResults.length}
+        </div>
+      </div>
+
+      <article className="container py-6 mx-auto">
+        <div className="grid grid-cols-1 mx-4 xl:mx-6 gap-4 justify-self-center">
+          {queryResults.map((result) => {
+            const fields = new Set<string>()
+            Object.keys(result.matchData.metadata).forEach((term) => {
+              Object.keys(result.matchData.metadata[term]).forEach((field) => {
+                fields.add(field)
+              })
+            })
+
+            const found = Array.from(fields).map((field, index) => (
+              <li
+                key={index}
+                className="inline mx-px px-3 py-1 mt-1 text-tag whitespace-nowrap font-semibold mr-2 bg-topics"
+              >
+                {field}
+              </li>
+            ))
+            const item = (
+              <React.Fragment key={result.ref}>
+                <div className="px-2 py-4 overflow-hidden bg-white border border-gray-200 rounded-md shadow-md h-full flex flex-col">
+                  <h2 className="capitalize">
+                    {db[result.ref].question || db[result.ref].slug}
+                  </h2>
+                  <ul className="grid grid-rows-1 justify-center md:block">
+                    <h2 className="font-sans text-black text-tag mb-1 font-extrabold">
+                      Match found in:
+                    </h2>
+                    {found}
+                  </ul>
+                  <h2 className="pt-3 mt-4 border-t-2 border-solid">
+                    <a className="btn-wide" href={`${siteUrl}/${result.ref}`}>
```

Comment:
```suggestion
                    <a className="btn-wide" href={`${siteUrl}/project/${result.ref}`}>
```

Reply:
I am adding "project" to each project's slug prior to this, since some pages won't have it (about page, contact page, etc)